support/Pipelines/EccentricityControl/InitialOrbitalParameters.py

```diff
@@ -0,0 +1,194 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+from typing import Optional, Sequence, Tuple
+
+import numpy as np
+from scipy.optimize import minimize
+
+logger = logging.getLogger(__name__)
+
+
+def initial_orbital_parameters(
+    mass_ratio: float,
```

Comment:
is there some fancy python thing like the maybe-some-day C++ Contracts that would allow you to say `mass_ratio: float(<1)(>0),`?

Reply:
Not built in. The values can only be constrained by docs and by asserts.

Reply:
Ah okay, thanks!